go/test/endtoend/vreplication/vdiff2_test.go

```diff
@@ -234,16 +235,52 @@ func testCLIErrors(t *testing.T, ksWorkflow, cells string) {
 
 func testDelete(t *testing.T, ksWorkflow, cells string) {
 	t.Run("Delete", func(t *testing.T) {
-		// test show verbose too as a side effect
+		// Let's be sure that we have at least 3 unique VDiffs.
+		// We have one record in the SHOW output per VDiff, per
+		// shard. So we want to get a count of the unique VDiffs
+		// by UUID.
+		uuidCount := func(uuids []gjson.Result) int64 {
```

Comment:
This can be replaced by 
```
		uuidCount := func(uuids []gjson.Result) int64 {
			seen := make(map[string]bool)
			for _, uuid := range uuids {
				seen[uuid.String()] = true
			}
			return int64(len(seen))
		}
```

Reply:
Yeah, that's a bit more compact. Will use that. Thanks!